src/content/index.md

```diff
@@ -6,17 +6,17 @@ path: /
 ---
 
 Hector is a little dinosaur  
-Who lives in the forest.  
+who lives in the forest.  
 He spends all his days eating flowers  
-And rest.
+and resting.
```

Comment:
Oh makes sense! But I might think of something else to make it rhyme with forest